Source/DataModels/Windows/EditorWindows/WindowConsole.cpp

```diff
@@ -69,10 +90,58 @@ void WindowConsole::DrawOptionsMenu()
 			consoleContents.clear();
 		}
 
+		DrawMaxLengthSelection();
+
 		ImGui::EndMenuBar();
 	}
 }
 
+void WindowConsole::DrawMaxLengthSelection()
+{
+	int numButtons = static_cast<int>(consoleLineLengths.size());
+	const char* limitText = "Limit console lines:";
+
+	// Align the buttons to the right
+	ImGuiStyle& style = ImGui::GetStyle();
+
+	float buttonWidth = ImGui::CalcTextSize("XXX").x + style.FramePadding.x * 2.0f;
+	float buttonsWidth = (buttonWidth + style.ItemSpacing.x) * numButtons;
```

Comment:
We can rename this variable to make it easier to differenciate from the `buttonWidth` one

```suggestion
	float buttonContainerWidth = (buttonWidth + style.ItemSpacing.x) * numButtons;
```